server/configuration.go

```diff
@@ -42,6 +43,9 @@ func (c *configuration) Clone() *configuration {
 
 // IsValid checks if all needed fields are set.
 func (c *configuration) IsValid() error {
+	if _, err := url.ParseRequestURI(c.GitlabURL); err != nil {
```

Comment:
Why `url.ParseRequestURI` instead of `url.Parse`?

Reply:
`url.Parse` can be [relative](https://godoc.org/net/url#Parse) : `/toto/tata` is valid